### PR TITLE
task 2 get reviews comeplete with forEach testing

### PR DESCRIPTION
**Pull request by roseannac21**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -30,4 +31,27 @@ describe("app tests", () => {
...
+          expect.objectContaining({title: expect.any(String), designer: expect.any(String), owner: expect.any(String), review_img_url: expect.any(String), review_body: expect.any(String), category: expect.any(String), created_at: expect.any(String), votes: expect.any(Number)});
+          expect(obj).toHaveProperty("title");
+          expect(obj).toHaveProperty("designer");
+          expect(obj).toHaveProperty("owner");
+          expect(obj).toHaveProperty("review_img_url");
+          expect(obj).toHaveProperty("review_body");
+          expect(obj).toHaveProperty("category");
+          expect(obj).toHaveProperty("created_at");
+          expect(obj).toHaveProperty("votes");
+        })
```
Comment (HeatherEver) [inline __tests__/app.test.js:53]: we've already tested above that the returned array contains an object containing all of these properties so `.toHaveProperty` is a little redundant here as it's already tested above

File: db/controller.js [hunk 2]
```diff
@@ -10,4 +10,14 @@ const getCategories = (request, response, next) => {
...
 
-module.exports = { getCategories };
\ No newline at end of file
+const getReviews = (request, response, next) => {
+
+  fetchReviews().then((reviews) => {
+    response.status(200).send({ reviews });
+  })
+  .catch((err) => {
+    console.log(err)
```
Comment (HeatherEver) [inline db/controller.js:19]: you are passing in 'next' to the function but not using it - something to think about when error handling :)

**Review by HeatherEver — APPROVED** [review]
Nice work Roseanna 👍 lovely testing and clean code :)

File: db/controller.js (hunk 2 above)
Comment (roseannac21) [reply db/controller.js:19]: cool, will add this in for my next pull request

**Pull request merged**